lib/App/DuckPAN/Cmd/Test.pm

```diff
@@ -6,20 +6,29 @@ with qw( App::DuckPAN::Cmd );
 
 use MooX::Options protect_argv => 0;
 
+option full => (
+	is          => 'ro',
+	lazy        => 1,
+	short       => 'f',
+	default     => sub { 0 },
+	doc         => 'run full test suite via dzil',
+);
+
 sub run {
-    my ( $self ) = @_;
+	my ( $self ) = @_;
 
-    my $ret = 0;
+	my $ret = 0;
 
-    if (-e 'dist.ini') {
-      $ret = system('dzil test');
-      $self->app->emit_error('Could not begin testing. Is Dist::Zilla installed?') if $ret == -1;
-    } else {
-      $self->app->emit_notice("Could not find dist.ini.");
-      $ret = system('prove -Ilib');
-    }
+	if ($self->full) {
+		$self->app->emit_error("Could not find dist.ini.") unless -e "dist.ini";
+		$ret = system("dzil test");
+		$self->app->emit_error("Could not begin testing. Is Dist::Zilla installed?") if $ret;
```

Comment:
I was just saying that there's no reason to catch the value and then test it.:
``` Perl
$self->app->emit_error("Could not begin testing. Is Dist::Zilla installed?") if $ret = system("dzil test");
```
Same with below.  Not a big deal, just more succinct.